### ISSUE TITLE
UI Truncation Clarification

### ISSUE THREAD

**Issue by shotnothing** [nus-cs2103-AY2425S1/forum#850]
Hi! Could I check if a GUI that, at the default size (i.e. fresh install, not fullscreen) has information truncated from a cropped result box with no scroll bar, is it still considered a valid bug?

The typical solutions would be to set minHeight of the main window, add a scrollbar or set the app to open in fullscreen by default. In the absence of these, would it be considered a UI bug?

**Comment by yadobler** [comment]
- how does it compare to the [tp-constraints for screen size](https://nus-cs2103-ay2425s1.github.io/website/admin/tp-constraints.html#constraint-screen-resolution)?
- "default size" as in, immediately when opened and is not "maximised"?
- "fullscreen" meaning "maximised"?

also
- not related but technically not "fresh install" since jar files are "portable" (it is also a constraint to not have anything installed)

---

it's a valid bug but i think your question is asking whether it is a "cosmetic" bug or an actual "functional" bug, is that right?

**Comment by shotnothing** [comment]
Yup, it works maximized (for the linked constraints) but the default window size (basically default `preferences.json` generated on fresh uhh start of app haha)  is that a certain box gets cropped so much the information inside isn't visible.

My question is if it is a valid bug since the workaround is to maximize the screen, even though there are alternative and what I feel are better solutions to this (e.g. minHeight) and that a user should be expected to use the app at the default dimensions, or at least have indication that they will need to maximize it, but I'm not sure...

**Comment by damithc** [comment]
@shotnothing If something is not visible in the GUI at app start up, that is definitely not ideal, and anything not ideal has room for improvement whether we call it a feature flaw or functionality bug. The harder part is to decide if it is NotInScope and the severity level.

On a related note, IIRC, AB3 'remembers' the last used window size. So, the user can set it to whichever size she wants and that size will 'stick' for future runs.

**Comment by shotnothing** [comment]
Alright, thank you for the clarification
